samples/sqltransport/multi-instance/Client/App.config

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+    <section name="MessageForwardingInCaseOfFaultConfig" type="NServiceBus.Config.MessageForwardingInCaseOfFaultConfig, NServiceBus.Core" />
+    <section name="UnicastBusConfig" type="NServiceBus.Config.UnicastBusConfig, NServiceBus.Core" />
+    <section name="AuditConfig" type="NServiceBus.Config.AuditConfig, NServiceBus.Core" />
+  </configSections>
+  <startup>
```

Comment:
the auto gen'd sections are documenting something that is not the main aim of any give sample. if you go down the path of tying to document things not related to the sample then it will become so bloated that it will obscure what you are trying to demonstrate